core/src/main/java/org/infinispan/xsite/OfflineStatus.java

```diff
@@ -74,11 +74,16 @@ public OfflineStatus(TakeOfflineConfiguration takeOfflineConfiguration, Supplier
 
    public void start() {
       globalState.addFilteredListener(this, this, null, Set.of(CacheEntryModified.class, CacheEntryCreated.class));
-      if (globalState.containsKey(key)) {
-         // it will receive the event with the up-to-date value
+      var existingStatus = globalState.get(key);
```

Comment:
In theory, it should not be blocked. The `CONFIG` cache should have finished the state transfer, and the reads are local in a replicated cache.

Reply:
> In theory, it should not be blocked.

Can we change to `compute` then? Or does ^ mean that you have found different in reality.

Reply:
`compute` will always use an RPC since it needs to be forwarded to the primary owner. Also, I'm unsure if the lambda serialization could cause any backward incompatibility. 

I didn't find any issues with this solution.